sky/shell/tracing_controller.cc

```diff
@@ -108,6 +121,8 @@ void TracingController::ManageObservatoryCallbacks(bool add) {
 void TracingController::StartTracing() {
   LOG(INFO) << "Starting trace";
 
```

Comment:
It feels like that concern should go next to the observatory handling bits.  Seems like it's a programming error (hence the assert) to call this StartTracing manually more than once?  Maybe not?  Donno.